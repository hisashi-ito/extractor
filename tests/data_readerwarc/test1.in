WARC/1.0
WARC-Type: conversion
WARC-Target-URI: http://statmt.org/
WARC-Date: 2017-07-20T12:39:49Z
WARC-Record-ID: <urn:uuid:4e2c9e9d-16ad-487d-a7a3-a7844df236c2>
WARC-Refers-To: <urn:uuid:acf277dd-0b09-44bb-b7dd-1429e7b6cdca>
WARC-Block-Digest: sha1:UZJYKZZJH3QI3RMVNKBAHMU3VKEFZGS2
Content-Type: text/plain
Content-Length: 6598

This website is dedicated to research in statistical machine translation, i.e. the translation of text from one human language to another by a computer that learned how to translate from vast amounts of translated text.
