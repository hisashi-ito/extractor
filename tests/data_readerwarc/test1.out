df6fa1abb58549287111ba8d776733e9 uri:http://statmt.org/
This website is dedicated to research in statistical machine translation, i.e. the translation of text from one human language to another by a computer that learned how to translate from vast amounts of translated text.
